samples/iterable_dataset/dataset_from_blob_list.py

```diff
@@ -0,0 +1,21 @@
+"""
+This sample demonstrates the way to create an iterable-style dataset from a blob list using the IterableBlobDataset class.
+"""
+
+# [START sample]
```

Comment:
If we are able to fit the docstring to one line, I'm wondering if we can just use the docstring quotes (e.g., `"""` as the sentinel value)? I think it would be worth to explore to possibly make it less awkward in having these `START` comments in all of these samples.